shapers/base.lua

```diff
@@ -67,8 +67,18 @@ function shaper:measureChar (char)
    local options = SILE.font.loadDefaults({})
    options.tracking = SILE.settings:get("shaper.tracking")
    local items = self:shapeToken(char, options)
-   if #items > 0 then
-      return { height = items[1].height, width = items[1].width, depth = items[1].depth }
+   SU.dump({ char, items })
+   if #items >= 1 then
+      local item = items[1]
+      if item.name == ".notdef" then
+         SU.warn(
```

Comment:
But should we really warn, or let the caller warn if needed?
Ex. in the zenkaku case below, we'll have a specific warning, to this one comes as extraneous, no?

Reply:
I reworked this, see previous comment for discussion points on throwing a warning or not.